tests/rustdoc/inline_local/staged-inline.rs

```diff
@@ -13,6 +13,6 @@ pub mod ffi {
 
 #[stable(feature = "v1", since = "1.0.0")]
 #[doc(inline)]
-//@ has "foo/struct.CStr.html" "//span[@class='sub-heading']/span[@class='since']" "1.0.0"
-//@ !has - "//span[@class='sub-heading']/span[@class='since']" "1.99.0"
+//@ has "foo/struct.CStr.html" "//span[@class='sub-heading']/span[@class='since']" "1.99.0"
+//@ !has - "//span[@class='sub-heading']/span[@class='since']" "1.0.0"
```

Comment:
This does not look good to me, because it causes docs to claim that std::ffi::CStr was available in 1.64.0. It just trades one bug for a different bug.

Instead, if you just want to fix this one problem, modify the propagate_stability code to ignore glob exports. It's reasonable to assume that anyone who re-exports one and only one item will write the correct stability marker on that re-export.

Reply:
@notriddle I have addressed the review comments. Please check